### DISCUSSION TITLE
Windows raises a security popup "Windows protected your PC" during installation

### DISCUSSION THREAD

**Discussion by QuestioningEspecialy** [setvisible/DownZemAll#182] [Q&A]
Windows 10 64bit
Tested v2.5.8, v3.0.0, v3.0.1, and v3.0.2 from their respective [release](https://github.com/setvisible/DownZemAll/releases) links.

Below is the blue popup I get for every v3.0.x version:
> Microsoft Defender SmartScreen prevented an unrecognized app from starting. Running this app might put your PC at risk.
> App: DownZemAll_x64_Setup.exe
> Publisher: Unknown publisher

[VirusTotal](https://www.virustotal.com/gui/file/d7a89abaad75bd25b9cf63b0e9ba4d68e6c9d7fbe52759e62485eb9cf7344a9c)'s v3.0.2 page says "SecureAge: Malicious" while Avira and NordVPN on my local didn't find anything wrong with it.
So~ what's up with the v3.0.x versions?

**Comment by setvisible** [comment]
Hi,

The first message from Windows Defender always happens to me too. I guess it's because I'm the first guy in the world to install it :)

For the second question, the "SecureAge" is probably a False Positive on the installer and "Uninstall.exe" too. Same with the "launcher.exe" which has been detected as containing a W32.Malware.Gen by "Webroot". I don't know why, but am happy to see that the majority of antimalware protection recognized it as a secure project. Maybe the SecurAge and Webroot software consider it as suspicious because the files are not signed (see message details: "File is not signed"). Indeed there are not signed.

More generally speaking, this project is safer than most of others: Indeed it uses 100% open-source libraries, plus the compiler is also free and open-source and verified by the community (GCC). This project is transparent, which is the best protection, and a good argument against malware suspicion.

Regards
